app/models/concerns/courses/edit_options/engineers_teach_physics_concern.rb

```diff
@@ -0,0 +1,16 @@
+module Courses
+  module EditOptions
+    module EngineersTeachPhysicsConcern
+      extend ActiveSupport::Concern
+      included do
+        # When changing anything here be sure to update the edit_options in the
+        # courses factory in publish-teacher-training:
+        #
+        # https://github.com/DFE-Digital/publish-teacher-training/blob/master/spec/factories/courses.rb
```

Comment:
Did you do it?

Reply:
I made a trait, but I can't see any `edit_options` in the courses factory?

Reply:
Bin the comments?

Reply:
Yep